ui/helpers/utils/notification.util.ts

```diff
@@ -381,7 +381,7 @@ export function getRpcUrlByChainId(chainId: HexChainId): string {
 
   // If rpc is found, return its URL. Otherwise, return a default URL based on the chainId.
   if (rpc) {
-    return rpc.rpcUrl;
+    return rpc.rpcEndpoints[0].url;
```

Comment:
I'm not sure whether its correct that this code determines RPC endpoints from the canonical ones in `FEATURED_RPCS`, instead of respecting the users custom rpc endpoints.  But replicating what it did before

Reply:
TY for these changes. This is consistent with the original code.

For visbility/context on this - our notifications team are not using the users RPCs for any specific transaction. We are only using the RPC to fetch some gas fee information when viewing some notifications.
Also notifications can come cross multiple chains (that a user may not be tracking).

These notification utils will be taken out into shared libraries (to be reused with mobile).

Reply:
This looks good, so resolving this comment. Our team will do any additional changes to keep this PR unblocked.